requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.12.4
```

Comment:
Here are some people exploring the same issue http://stackoverflow.com/questions/9232568/identifying-the-dependency-relationship-for-python-packages-installed-with-pip

On the bright side, specifying everything explicitly in requirements.txt seems to be a best practice: "Requirement files are mostly flat. Maybe MyApp requires Framework, and Framework requires Library. I encourage you to still list all these in a single requirement file" https://pip.readthedocs.io/en/1.1/requirements.html